framework/src/utils/SlepcSupport.C

```diff
@@ -108,6 +108,13 @@ getSlepcEigenProblemValidParams()
 
   params.addParam<Real>("l_abs_tol", 1e-50, "Absolute Tolerances for Linear Solver");
 
+  // When the eigenvalue problems is reformed as a coupled nonlinear system,
+  // we use part of Jacobian as the preconditioning matrix.
+  // Because the difference between the Jacobian and the preconditioning matrix is not small,
+  // the linear solver KSP can not reduce the residual much. After several tests,
+  // we find 1e-2 is a reasonable choice.
+  params.set<Real>("l_tol") = 1e-2;
```

Comment:
Oh, Hang on. I thought you were inheriting parameters from the solver and overriding this value. You aren't though, you are starting from empty parameters. In this case, I'd prefer to see you use the addParam() call instead of set. The reason is that we won't get the proper meta-data out for Peacock, or MooseDocs since the documentation string won't exist.

Reply:
I was intending to inherit parameters.

Reply:
Now it inherits  from `Excutioner`